visual-tests/imports/helpers.js

```diff
@@ -0,0 +1,67 @@
+export const helpers = {
+  defaultHOTWrapper: 'js',
+
+  screenshotsDirectory: './tests/screenshots',
```

Comment:
```suggestion
  screenshotsDirectory: './screenshots',
```
Can I propose changing the directory where the screenshots are saved? Currently, they are saved in the same place as the test files. I think the better place for that would be the root of the project. There are already created other non-comittable dirctories as `test-results` or `playwright-report`.

Reply:
Sure - changed.

Reply:
Don't forget to update the `.gitignore` file :)

Reply:
yes, it should be already updated, but, at least temporary, I left there previous path also - just in case if somebody already have screenshots on local machine (e.g. me - I'm not sure who else was playing with this testing already)